packages/api/src/store/queue.ts

```diff
@@ -77,8 +79,14 @@ export class RabbitQueue implements Queue {
       json: true,
       setup: async (channel: Channel) => {
         await Promise.all([
-          channel.assertQueue(QUEUES.events, { durable: true }),
-          channel.assertQueue(QUEUES.webhooks, { durable: true }),
+          channel.assertQueue(QUEUES.events, {
+            arguments: { "x-queue-type": "quorum" },
+            durable: true,
```

Comment:
Little thing: Do these quorum queues need to be specified as durable? As far as I can understand quorum queues are durable by default and do not support non-durability

Reply:
You are right, quorum queues cannot be non-durable. I ended up keeping that here to have it set explicitly here, but if it might create confusion I'll just remove it, which makes sense and is cleaner.